sonar-project.properties

```diff
@@ -0,0 +1,17 @@
+sonar.organization=hootenanny
+sonar.projectKey=hoot
+sonar.exclusions=**/*.pb.cc,**/*.pb.h,**/*.sql,**/*.o,**/*.gcda,**/*.gcno,**/*.qmake,**/*.pro,**/VersionDefines.h,**/ConfigDefaults.h,**/ConfigOptions.h,**/TgsConfig.h
+sonar.github.repository=ngageoint/hootenanny
+sonar.host.url=https://sonarcloud.io
+sonar.sources=./hoot-core/src/main,./hoot-js/src/main,./hoot-rnd/src/main,./tbs/src/main,./tgs/src/main
+
+sonar.issue.ignore.multicriteria=hoot1,hoot2,hoot3
+# Only the command classes should output to standard out
+sonar.issue.ignore.multicriteria.hoot1.ruleKey=cpp:S106
+sonar.issue.ignore.multicriteria.hoot1.resourceKey=**/hoot/core/cmd/*
+sonar.issue.ignore.multicriteria.hoot2.ruleKey=cpp:S106
+sonar.issue.ignore.multicriteria.hoot2.resourceKey=**/hoot/rnd/cmd/*
```

Comment:
I tried combining rules `hoot1` and `hoot2` with the `resourceKey` as a comma-separated list but that only picked up the second wildcard.  Splitting the rule in to allows you to exclude the rule in two different directories.